nucypher/cli/painting.py

```diff
@@ -418,10 +446,26 @@ def paint_staged_stake_division(emitter,
                        stake_value=target_value,
                        lock_periods=new_duration_periods,
                        start_period=original_stake.start_period,
-                       end_period=new_end_period,
+                       unlock_period=new_end_period,
                        division_message=division_message)
 
 
+def paint_accounts(emitter, balances):
+    rows = list()
+    max_eth_len, max_nu_len = 0, 0
+    for address, balances in sorted(balances.items()):
+        eth = str(Web3.fromWei(balances['ETH'], 'ether')) + " ETH"
+        nu = str(NU.from_nunits(balances['NU']))
```

Comment:
Do we want to append " NU" like we append " ETH" in the line above?

Reply:
The `NU.__str__()` method already does that by default.